<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">_</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="margin">
     <number>10</number>
    </property>
    <item>
     <widget class="QGroupBox" name="gb1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Time elapsed</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>-1</number>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="time_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string notr="true">00:00</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gb2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Configurations</string>
      </property>
      <layout class="QGridLayout" name="gridChildLayout">
       <property name="margin">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Set timer</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="sound_check">
         <property name="text">
          <string>Disable sound</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="time_sb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string notr="true"/>
         </property>
         <property name="prefix">
          <string notr="true"/>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>120</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="blink_check">
         <property name="text">
          <string>Disable icon blink</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="notify_check">
         <property name="text">
          <string>Disable notifications</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="reset_check">
         <property name="text">
          <string>Reset timer a manual start</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>time_sb</zorder>
      <zorder>label</zorder>
      <zorder>sound_check</zorder>
      <zorder>blink_check</zorder>
      <zorder>notify_check</zorder>
      <zorder>reset_check</zorder>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gb3">
      <property name="title">
       <string>Something to remember</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>218</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAlarm">
    <property name="title">
     <string>&amp;Alarm</string>
    </property>
    <addaction name="action_Start"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionAbout_qt"/>
   </widget>
   <addaction name="menuAlarm"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_Start">
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>A&amp;bout...</string>
   </property>
  </action>
  <action name="actionAbout_qt">
   <property name="text">
    <string>About &amp;qt...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
